main.js

```diff
@@ -21,4 +23,356 @@ hamImg.addEventListener('click', menu);
+            <button class = "seeProBtn" type="button" autofocus>See Project </button>
+          </article>
+          <article class="desktop-card-5 desk-card">
+            <div class="card-text">
+              <h3>
+               ${projectInfo[5].title}
+              </h3>
+              <p>
+                ${projectInfo[5].description}
+              </p>
+              <ul>
+                ${getTechList(projectInfo[5].technologies)}
+              </ul>
+            </div>
+            <button class = "seeProBtn" type="button">See Project </button>
+          </article>
+          <article class="desktop-card-6 desk-card">
+            <div class="card-text">
+              <h3> ${projectInfo[6].title}</h3>
+              <p>
+                ${projectInfo[6].description}
+              </p>
+              <ul>
+               ${getTechList(projectInfo[6].technologies)}
+              </ul>
+            </div>
+            <button class = "seeProBtn" type="button">See Project </button>
+          </article>
+        </div>
+      `;
+}
+
+loadData();
+/* project data load end */
+/* pop up window start */
+const popUpWinFirst = document.querySelector('.popUpWinFirst');
+const popUpWinSecond = document.querySelector('.popUpWinSecond');
+
+function popUpOPenClose(section) {
+  section.classList.add('popUpOpen');
+  document.querySelector('#main').style.filter = 'blur(5px)';
+  document.querySelector('.hightNone').style.display = 'none';
+  document.querySelector('#crossPop').addEventListener('click', () => {
+    section.classList.remove('popUpOpen');
+    document.querySelector('#main').style.filter = 'none';
+    document.querySelector('.hightNone').style.display = 'block';
+  });
+}
+
+function loadDataPopFirst() {
+  popUpWinFirst.innerHTML = `
+     <div class="projectPreview">
+        <div class="head">
+          <h2>${projectInfo[0].title}</h2>
+          <img id ="crossPop" src="icons/cross.png" alt="cross">
+        </div>
+        <ul class="technologiesPreview">
+          ${getTechList(projectInfo[0].technologies)}
+        </ul>
+        <div class="project_elements">
+          <img class="projectImg" src="${projectInfo[0].imagePop}" alt="pop up image" />
+          <div class="details_container">
+            <div class="projectDescription">
+              <p>
+                ${projectInfo[0].description}
+              </p>
+            </div>
+            <div class="projectButtons">
+              <button class="cardButtons" href="${projectInfo[0].link}">
+                <span>See Live</span>
+                <img src="icons/seeLiveIcon.svg" alt="live" />
+              </button>
+              <button class="cardButtons" href="${projectInfo[0].github}">
+                <span>See Source</span>
+                <img src="icons/whiteGitHubIcon.png" alt="Source" />
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      `;
+
+  popUpOPenClose(popUpWinFirst);
+}
+
```

Comment:
You have done a great job implementing the popup functionality👍 but there are still some improvements we can make so our project looks just like the design😉😉 
Your pop-up currently looks a bit different from the given design kindly consider the spacing, image size, font-property and size of the container apportioned to the project description as you make your changes🤗🤗

your popup(mobile)
<img width="268" alt="your__mobilepopup" src="https://user-images.githubusercontent.com/64643995/217586901-ea4e45b4-174f-49d7-bb27-6341f9fd5956.png">

your popup desktop
<img width="878" alt="your_desktoppopup" src="https://user-images.githubusercontent.com/64643995/217587011-84cc679e-ea91-47b3-8ea4-b53e593f7b1f.png">

design(desktop)
<img width="433" alt="design_popup" src="https://user-images.githubusercontent.com/64643995/217587106-59316775-45df-4f26-ba76-4abfa945f521.png">

design(mobile)
<img width="169" alt="mobile_popup" src="https://user-images.githubusercontent.com/64643995/217587795-2bde8167-d6de-4fce-8e52-e9d0027271c7.png">

don't forget, it is a requirement in this milestone that you stick to the given design. kindly make comparison and make the necessary changes😉😉😉
<img width="679" alt="popup_sticitodesign" src="https://user-images.githubusercontent.com/64643995/217588862-8e939377-6ec6-4f47-b5cb-ef63a8d38bc6.png">